src/main/scripts/import_objects.groovy

```diff
@@ -189,6 +189,7 @@ else {
     }
 
     if (foundErrors) {
+		script.delete()
```

Comment:
Is there a reason why you would want to keep the script on success? 

If it contains password, it likely should be deleted regardless of success or failure. If so, the safer approach would be using a "script.deleteOnExit()" immediately after it's creation on line 109. That way, if if fails oddly in between 109 and 192, the script will still get deleted.

If you believe users other users would like to see the script file after completion, then maybe an assumed default deleteOnExit and checkBox property to save the script would be best.